xbmc/dialogs/GUIDialogBusy.cpp

```diff
@@ -109,7 +109,7 @@ CGUIDialogBusy::CGUIDialogBusy(void)
 
 CGUIDialogBusy::~CGUIDialogBusy(void) = default;
 
-void CGUIDialogBusy::Open_Internal(const std::string &param /* = "" */)
+void CGUIDialogBusy::Open_Internal(bool bProcessRenderLoop, const std::string &param /* = "" */)
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types